draft-irtf-cfrg-hpke.md

```diff
@@ -367,8 +367,8 @@ function specification for DHKEMs defined in this document.
 
 ~~~
 def ExtractAndExpand(dh, kemContext):
-  prk = LabeledExtract(zero(0), concat(I2OSP(kem_id, 2), "dh"), dh)
-  return LabeledExpand(prk, concat(I2OSP(kem_id, 2), "prk"), kemContext, Nzz)
+  prk = LabeledExtract(zero(0), concat(I2OSP(kem_id, 2), "prk"), dh)
+  return LabeledExpand(prk, concat(I2OSP(kem_id, 2), "zz"), kemContext, Nzz)
 
```

Comment:
Maybe for the sake of consistency you namespace these as well. `eae_prk` and `eae_zz`

Reply:
I changed to `eae_prk`, but left zz untouched, because zz is unique, and we also did not namespace sk (because it is unique as well).